### PR TITLE
test: show bad location in empty interpolation error

### PR DESCRIPTION
**Pull request by anmonteiro**

### CODE REVIEW

File: test/unit-tests/ounit_unicode_tests.ml [hunk 1]
```diff
@@ -172,15 +172,6 @@ let suites =
                    Invalid_syntax_of_var " (" ) ->
                OUnit.assert_bool __LOC__ true
            | _ -> OUnit.assert_bool __LOC__ false );
-         ( __LOC__ >:: fun _ ->
```
Comment (anmonteiro) [inline test/unit-tests/ounit_unicode_tests.ml:175]: this test was duplicated with the one I changed below.

File: test/blackbox-tests/utf8-string-interp.t [hunk 2]
```diff
@@ -0,0 +1,22 @@
+
+
+  $ . ./setup.sh
+  $ cat > x.ml <<EOF
+  > let x = {j| Hello, \$()|j}
+  > EOF
+  $ melc -ppx melppx x.ml
+  File "x.ml", line 1, characters 20-23:
+  1 | let x = {j| Hello, $()|j}
+                          ^^^
```
Comment (jchavarri) [inline test/blackbox-tests/utf8-string-interp.t:10]: Is this on the right location? Seems it's picking `()|` rather than `$()`.

Comment (jchavarri) [reply test/blackbox-tests/utf8-string-interp.t:10]: Sorry, after reading the title, it's unclear to me if the PR was created to showcase this bad location or to fix it.

Comment (anmonteiro) [reply test/blackbox-tests/utf8-string-interp.t:10]: oh good point, I guess we could move it backwards yet one more char.

The [first commit](https://github.com/melange-re/melange/pull/888/commits/f4f814fa8b90e63c3631fe916fa753ed3c2991cc) shows the wrong location, and the [2nd commit](https://github.com/melange-re/melange/pull/888/commits/3c41146f314f895177b76fc64202a966b7214c7a) fixes it.

**Pull request merged**